Library/Homebrew/formula_installer.rb

```diff
@@ -1546,3 +1546,5 @@ def puts_requirement_messages
     $stderr.puts @requirement_messages
   end
 end
+
+require "extend/os/formula_installer"
```

Comment:
Are the methods even defined through this `require` used anywhere?

Seems to me that we can just remove them.

Reply:
Only here AFAICT: https://github.com/Homebrew/brew/blob/6a0db50/Library/Homebrew/formula_installer.rb#L270